stylesheet.scss

```diff
@@ -14,6 +14,15 @@
   text-shadow: black 0px 2px 2px;
 }
 
+// Page Indicators
+
+.page-indicator {
+  padding: 15px 12px;
+  .page-indicator-icon {
+    box-shadow: 0 0 4px 3px black;
```

Comment:
Have you tried this shadow on a less noisy wallpaper? I would probably not bother adding the shadow (as in 3.8) but definitely not a blocker.

Reply:
It almost doesn't make any difference either on noisy and smooth wallpapers, but between terrible and less terrible, I prefer to go with the latter :)